beacon_node/http_api/src/lib.rs

```diff
@@ -2623,6 +2625,50 @@ pub fn serve<T: BeaconChainTypes>(
             },
         );
 
+    // GET debug/beacon/ssz/{state_id}/
```

Comment:
Is it possible to have the endpoint take an array of state ids and paths. It would be useful since we can need to generate up to 3 SSZ proofs at once

Reply:
discussed offline. Original motivation is to batch to make the query faster. But it's just fast enough now so we don't need batching right now.

latency for testing query 0m0.008s

Reply:
Added more info in the ticket https://linear.app/intrinsictech/issue/INT-2065/beacon-chain-httpgrpc-api-for-beacon-state

historical slots will still be slow. I'm not sure batching will help to improve a lot. If we need good latency for fuzz testing, I think we should set up some testing node with large enough cache

Reply:
Yup, sounds good. Latency is solid for the keystore use case. For fuzzing, we can figure out the node requirements once it comes time to do that since I'll have a better idea of the ranges I would want.